content/concepts/decentralized-exchange/ticksize.md

```diff
@@ -13,8 +13,11 @@ When an issuer enables, disables, or changes the `TickSize`, Offers that were pl
 ## See Also
 
 - [Dev Blog: Introducing the TickSize Amendment](https://xrpl.org/blog/2017/ticksize-voting.html#ticksize-amendment-overview)
+- **References:**
+    - [AccountSet transaction][]
+    - [book_offers method][]
+    - [OfferCreate transaction][]
```

Comment:
Looks like there's a TODO to add link definitions, but want to call out that these come out as plaintext in the local build.

Reply:
It's not a TODO. The link definitions are there right under that line (just past what's hidden by the GitHub diff) and they're looking fine to me

Reply:
I know, I see the definitions there, too, which is why I thought it was a TODO. The definition's also in the source file. But seeing the same thing:

<img width="486" alt="Screen Shot 2019-08-12 at 8 34 23 PM" src="https://user-images.githubusercontent.com/8494827/62913430-d922bd80-bd40-11e9-82cb-0337801b5e5e.png">


Ah, I found the problem. Line 19 should be:

> `- [OfferCreate transaction][]`

For some reason, my local file had `[CreateOffer transaction][]`.